ntbs-service-unit-tests/Models/Entities/NotificationTest.cs

```diff
@@ -67,5 +69,26 @@ public void NotificationSetsShouldBeClosedCorrectly(DateTime treatmentOutcomeDat
             // Assert
             Assert.Equal(expectedValue, notification.ShouldBeClosed());
         }
+
+        [Fact]
+        public void ShouldBeClosedThrowsExceptionWhenOutcomeNotLoaded()
+        {
+            // Arrange
+            var notification = new Notification
+            {
+                NotificationStatus = NotificationStatus.Notified,
+                TreatmentEvents = new List<TreatmentEvent> {
+                    new TreatmentEvent
+                    {
+                        TreatmentEventType = TreatmentEventType.TreatmentOutcome,
+                        TreatmentOutcomeId = 7,
+                        EventDate = new DateTime(2003, 4, 15)
```

Comment:
I think it might be worth explicitly setting the `TreatmentOutcome` to null here